features/docs/defining_steps/ambiguous_steps.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Ambiguous Steps
+ 
+  Scenario: Ambiguous steps
+
+    Given a file named "features/ambiguous.feature" with:
+    """
+    Feature:
+
+      Scenario: test 1
```

Comment:
I tend to not even bother naming scenarios used as tests unless it adds something to the documentation. In this case I suggest you can leave it blank.